### PR TITLE
Added Footer to Website

### PR DESCRIPTION
**Pull request by LoganAJones**
I have added a footer to the website with the help of Bootstrap. I had to look into Bootstrap and how their stylesheets and elements work. It was similar to Foundation which I have worked with previously. I added columns to put Links to the Repository a the image sources we were pulling from.

### CODE REVIEW

File: src/components/App.vue [hunk 1]
```diff
@@ -1,7 +1,47 @@
...
+          <!-- Grid row -->
+        </div>
+        <!-- Footer Text -->
+        <!-- Copyright -->
+        <div class=" text-uppercase footer-copyright text-center font-weight-bold text-primary py-3">We Love Star Wars
+        </div>
+        <!-- Copyright -->
+      </footer>
+      <!-- Footer -->
+
```
Comment (alexkramer) [inline src/components/App.vue]: Its good to be consistent with your new lines. Here you have one but there is no newline between the footer open and the close of the main tag.

File: src/components/App.vue [hunk 2]
```diff
@@ -1,7 +1,47 @@
...
     <main>
       <router-view />
     </main>
+    <!-- Footer -->
+    <footer class="page-footer font-small px-10 red pt-4 bg-light">
+        <!-- Footer Text -->
+        <div class="container-fluid text-center text-md-left">
+          <!-- Grid row -->
+          <div class="row">
+            <!-- Grid column -->
```
Comment (mauracwarner) [inline src/components/App.vue]: Is the rest of the code commented with this level of detail?  A "Grid row" comment on a `div` with a class of `"row"` seems redundant to me.  If that's the convention throughout the project then it should be followed, but if not, I don't think all these comments are necessary.

File: src/components/App.vue [hunk 3]
```diff
@@ -3,5 +3,26 @@
...
+              <div class="text-center">
+              <h5 class="text-uppercase text-center font-weight-bold text-white bg-secondary">Our Repository</h5>
+              <a href="https://github.com/binxhealth/force-vue-awakens"> GitHub Repo</a>
+              </div>
+            </div>
+            <hr class="clearfix w-100 d-md-none pb-3">
+            <div class="col-md-6 mb-md-0 mb-3">
+              <div class="text-center bg-light">
+              <h5 class="text-uppercase text-center font-weight-bold text-white bg-secondary">Pictures</h5>
+              <a href="https://starwars-visualguide.com/#/"> starwars-visualguide.com</a>
```
Comment (alexkramer) [inline src/components/App.vue]:
```suggestion
              <a href="https://starwars-visualguide.com/#/">starwars-visualguide.com</a>
```

**Comment by alexkramer** [comment]
WIll close this PR since it will be overcome by the next PR which will integrate a base layout template.

**Pull request closed without merging**